Tests/SourceKitLSPTests/DocumentSymbolTests.swift

```diff
@@ -299,23 +299,23 @@ final class DocumentSymbolTests: XCTestCase {
   }
 
   func testMultiplePatternsInVariable() async throws {
-    try await assertDocumentSymbols("1️⃣var 2️⃣varA3️⃣: Int, 4️⃣varB5️⃣ = 06️⃣") { positions in
+    try await assertDocumentSymbols("var 1️⃣varA2️⃣: Int,3️⃣ 4️⃣varB5️⃣ = 06️⃣") { positions in
       [
         DocumentSymbol(
           name: "varA",
-          detail: "Int",
+          detail: nil,
```

Comment:
Is `detail` always `nil` now?

Reply:
Yes. We could try to get the type from a variable declaration if it is explicitly specified but I think it’s more confusing if you have it for `let x: Int = 1` but not for `let x = 1`, so I chose to just always set it to `nil`.